### DISCUSSION TITLE
Syncing problem between Keepass2Android and Keepass XC

### DISCUSSION THREAD

**Discussion by drunkenslang** [keepassxreboot/keepassxc#9044] [Q&A] [answered]
I'm running KeepassXC on windows and using Keepass2Android.  The db is on google drive secure using a keyfile and Yubikey w/ challenge response.  It all works great except if I update or add a password on K2A while KeepassXC is open on Windows, KeepassXC attempts to reload the change but if I'm not there w/ my Yubikey ready right away the change doesn't get recognized.  And then K2A ends up pulling the old version of the DB from XC.  

What I'm wondering is, is there a way to force a reload from the db changes that K2A made?  Or maybe a way to make a prompt based on time stamp as to which version has been edited last?  I see Keeshare is likely the solution but K2A doesn't have that.

As long as XC isn't running at the time on my PC then there's no problem but that's not really practical.

Any help would be appreciated.

Thanks

**Comment by droidmonkey** [comment, marked as answer]
See #5290. For now you need to lock your database on desktop in order to not lose data.
